tests/phpunit/tests/blocks/getBlockTemplates.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @group blocks
+ *
+ * @covers ::get_block_templates
+ */
+class Tests_Blocks_GetBlockTemplates extends WP_UnitTestCase {
+
+	const TEST_THEME = 'block-theme';
+
+	/**
+	 * @var WP_Post
+	 */
+	private static $template;
+
+	/**
+	 * @var WP_Post
+	 */
+	private static $template_part;
+
+	public static function wpSetUpBeforeClass() {
```

Comment:
[Per the Core Handbook](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-fixtures), should these set up/tear down methods be updated to their `snake_case` alternates for better compatibility?

Reply:
Fixed in 66946f6df5fe9b950503aa38df30c416f6ed2f30

Reply:
Just to clarify, that handbook paragraph specifically applies to the native PHPUnit methods that need to be replaced with  WordPress' `snake_case` counterparts, as the tests would not pass across all supported PHP and PHPUnit versions otherwise:
* `setUpBeforeClass()` → `set_up_before_class()`
* `tearDownAfterClass()` → `tear_down_after_class()`


It does not apply to WordPress' own methods (note the `wp` prefix):
* `wpSetUpBeforeClass()`
* `wpTearDownAfterClass()`

These are fine to use and are used in many other tests, much more often than the previous methods. I would actually like to standardize on these across the test suite for consistency, since they are already used in the majority of tests.

Related: [#55918](https://core.trac.wordpress.org/ticket/55918)